src/style.css

```
@@ -1 +1,29 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('../src/images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url('../src/images/star-active.svg');
```

```suggestion
  background-image: url(images/star-active.svg);
```
Try to update path, remove backstop_data folter and rerun tests